Xcode/BuildSystemIntegration.md

```diff
@@ -0,0 +1,43 @@
+## Build system integration
+
+In Xcode, there are 2 build systems: Legacy and XCBuild.
+
+As of Xcode 11, XCBuild is the default build system selected by Xcode. As these
```

Comment:
This statement is not correct, I wanted to say "On Xcode 11" . I don't know off the top of my head when this was a actually defaulted on.

Reply:
I _think_ it was Xcode 10?